contact.html

```diff
@@ -11,16 +11,21 @@
         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
         <link href="https://fonts.googleapis.com/css2?family=Bungee&display=swap" rel="stylesheet">
 
+        <!-- Tite Icon -->
+        <link rel="apple-touch-icon" sizes="180x180" href="favicon_io/apple-touch-icon.png">
+        <link rel="icon" type="image/png" sizes="32x32" href="favicon_io/favicon-32x32.png">
+        <link rel="icon" type="image/png" sizes="16x16" href="favicon_io/favicon-16x16.png">
+        <link rel="manifest" href="favicon_io/favicon_io/site.webmanifest">
     </head>
 
     <body>
 
         <!-- Navigation Bar -->
         <ul class="nav-bar">
             <div class="left">
-                <li><a href="./index.html">Home</a></li>
-                <li><a href="./experience.html">Experience</a></li>
-                <li><a class="active" href="./contact.html">Contact</a></li>
+                <li><a href="/">Home</a></li>
+                <li><a href="/experience.html">Experience</a></li>
+                <li><a class="active" href="/contact.html">Contact</a></li>
```

Comment:
It's a nit, but when you go back home, instead of the omnibar (searchbar) displaying `ameliakemp.dev`, it displays `ameliakemp.dev/index.html` which is uglier. With this change it won't.